### PR TITLE
Updated slack link to OSDC telegram link

### PR DESCRIPTION
**Pull request by codekhal**

### CODE REVIEW

**Comment by saranshbarua** [comment]
@codekhal you need to change the styled component SlackLink to TelegramLink too

https://github.com/osdc/osdc.github.io/blob/1ecd60883682375ac311c90e6c870957d0851da8/src/pages/index.js#L330

**Review by codekhal — COMMENTED** [review]
Does that solve the issue @saranshbarua

File: src/pages/index.js [hunk 1]
```diff
@@ -489,10 +489,10 @@ export default () => <Container>
           <span>GITHUB</span>
          </GithubLink>
       </Alink>
-      <Alink href="https://join.slack.com/t/jiit-lug/shared_invite/enQtNDI0MjM5NzcxNTM5LWY4M2U1YmFjMDYxZTZhMzQxMGIwN2QwOWUzZTQxNjAyYjFmMTQyMWEzYzkxZTkyMzQxOTNiNTFhYTRlYmMzOTY" target="_blank">
-        <SlackLink>
-          <span>SLACK</span>
-        </SlackLink>
+      <Alink href="https://t.me/jiitosdc" target="_blank">
+        <TelegramLink>
+          <span>Telegram</span>
```
Comment (saranshbarua) [inline src/pages/index.js]: Great work, a little change will make it perfect.
```suggestion
          <span>TELEGRAM</span>
```

**Review by saranshbarua — CHANGES_REQUESTED** [review]

**Review by ryzokuken — APPROVED** [review]
LGTM after Saransh's comment is addressed.

**Comment by ryzokuken** [comment]
Dismissing and merging.

**Review by saranshbarua — APPROVED** [review]

**Comment by saranshbarua** [comment]
@codekhal smooth work! Congratulations on your first successful contribution to OSDC 🎉

**Pull request merged**